app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -1,86 +1,222 @@
-a:visited { color: #484357; }
-
-.flash.success, .flash.error, .flash.inbox { background-color: #CDC7E7; }
-.pagination .current { background-color: #9692A8; }
-.pagination a { color: #F3F3F3; }
-.pagination a:visited { color: #C6C6E5; }
-.pagination .disabled { color: #58546B; }
-.pagination a:hover { background-color: #9498AC; }
+
+$font_color_selectionpopup: #DADADA;
+$bg_color_selectionpopup: #6F658D; // selection popups like character/alias on posts#editor
+$bg_color_expander: #D8D8D8; // expander on posts
+
+.selector-popup { background-color: $bg_color_selectionpopup; color: $font_color_selectionpopup; }
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_post_icon: #9498AC;
+$bg_color_icon_selector: #9692A8;
+$font_color_icon_selector: #DADADA;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; color: $font_color_icon_selector; }
+
+$bg_color_post_character: #645F8A;
+$bg_color_post_screenname: #645F8A;
+$bg_color_post_author: #42416E;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in
+// TODO: Should these flip on starries too?
+$font_color_post_info: #F3F3F3;
+
+.post-character { font-weight: normal; background-color: $bg_color_post_character; }
+.post-character, .post-character a, .post-character a:visited, { color: $font_color_post_info; }
+.post-screenname { font-weight: normal; background-color: $bg_color_post_screenname; }
+.post-screenname, .post-screenname a, .post-screenname a:visited { color: $font_color_link_visited; font-style: italic; } // TODO: clean this up when messages are fixed to not use it
+.post-author { background-color: $bg_color_post_author; font-weight: normal; } // TODO: font weights different from starrydark
+.post-author, .post-author a, .post-author a:visited { color: $font_color_post_info; }
+
+$font_color_paginate_link: #F3F3F3;
+$bg_color_paginate_link_hover: #9498AC;
+$font_color_page_visited: $font_color_link_visited;
+$bg_color_page_current: #9692A8;
+$font_color_page_disabled: #58546B;
+
+.pagination .current { background-color: $bg_color_page_current; }
+.pagination a { color: $font_color_paginate_link; }
+.pagination a:visited { color: $font_color_page_visited; }
+.pagination .disabled { color: $font_color_page_disabled; }
+.pagination a:hover { background-color: $bg_color_paginate_link_hover; }
+
+/* - icons */
+//$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #D8D8D8;
+$bg_color_icon_credit: #D8D8D8;
+
+/*.selected-icon { border-color: #409CFF; }*/ // TODO: Why is this commented out?
+.replace-icon, .icons-box { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+#character-icon-selector, .time-loaded { color: #DADADA; }
```

Comment:
(I think this isn't in the original and might break something?)

Reply:
...huh. how did it get there then?

Reply:
It's in some other starry – I guess because of that? (It's due to The Font Color Issue I Am Yet To Resolve.)